platform/mbed_wait_api.h

```diff
@@ -77,8 +78,9 @@ void wait_ms(int ms);
  *  @param us the whole number of microseconds to wait
  *
  *  @note
- *    If the RTOS is present, this function always spins to get the exact number of microseconds,
- *    which potentially affects power (such as preventing deep sleep) and multithread performance.
+ *    If the RTOS is present, this function spins to get the exact number of microseconds,
+ *    (for 1/100th of sec) which potentially affects power (such as preventing deep sleep) and 
+ *    multithread performance. You can avoid it by using ThisThread::sleep_for() / wait_ms()
```

Comment:
Isn't this true even if the RTOS is not present?

Reply:
If the RTOS is present, this function spins to get the exact number of microseconds - This is true if RTOS not present